src/schema/v2/auction_result.ts

```diff
@@ -250,13 +257,21 @@ const AuctionResultType = new GraphQLObjectType<any, ResolverContext>({
                 return null
               }
 
-              const { symbol, subunit_to_unit } = currencyCodes[
-                currency.toLowerCase()
-              ]
-              let display
+              const currency_map = currencyCodes[currency.toLowerCase()]
+              let symbol
+              let subunit_to_unit
+              if (currency_map) {
+                symbol = currency_map.symbol
+                subunit_to_unit = currency_map.subunit_to_unit
+              } else {
+                return null
+              }
 
+              let display
               if (indexOf(symbolOnly, currency) === -1) {
                 display = currency
+              } else {
+                return null
```

Comment:
Could we add a test here just to catch things in the future?

Reply:
We should maybe as well do `Sentry.captureException("missing currency", currency)` or whatever way to know about missing currencies in the future.